### PR TITLE
Gobuster dir, bbot sub-domain and AJAX

### PR DESCRIPTION
**Pull request by ph1ne4s**
Added new tools in Application discovery and subdomain takeover.
Resolved issue #1235 .
Addes AJAX vuln. resources.

## Types of changes

- [ x] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other

### Checklist:

- [ x] My code follows the code style (modified PEP8) of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: owtf/managers/resource.py
Comment (ph1ne4s) [inline owtf/managers/resource.py:1]: Resolves #1235

File: owtf/managers/resource.py [hunk 1]
```diff
@@ -105,7 +105,7 @@ def get_resources_from_file(resource_file):
         resource_file, "r"
     ).read().splitlines()  # To remove stupid '\n' at the end
     for line in config_file:
-        if "#" == line[0]:
+        if line.startswith("#"):
```
Comment (viyatb) [inline owtf/managers/resource.py:108]: `line` here is a list not a string

**Review by viyatb — CHANGES_REQUESTED** [review]
minor nit, everything else looks good

File: owtf/managers/resource.py (hunk 1 above)
Comment (ph1ne4s) [reply owtf/managers/resource.py:108]: Actually `line` is a string inside `config_file` which is a list, and so, `startwith` function is running fine without error.
str

Comment (ph1ne4s) [reply owtf/managers/resource.py:108]: Actually `line` is a string inside `config_file` which is a list, and so, `startwith` function is running fine without error.
str

**Review by viyatb — APPROVED** [review]

**Pull request merged**